### PR TITLE
docs: Fixed typo(s) & links

### PR DESCRIPTION
**Pull request by LorenzNickel**
Fixed two typos, updated links

## Description

Fixed one bigger and one minor typo, updated the links to the other SDKs.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [x] I have read the **README** document
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

## License

- [x] I confirm that this pull request can be released under the Apache 2 license

[issues]: https://github.com/alexa/alexa-skills-kit-sdk-for-nodejs/issues
[license]: http://aws.amazon.com/apache2.0/
[cla]: http://en.wikipedia.org/wiki/Contributor_License_Agreement

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,9 +95,9 @@ Sample skill that shows how to request and access the configured address in the
 Project that demonstrates how to use the audio player for skills.
 
 ## Other Language Alexa Skills Kit SDKs
-<a href="https://github.com/amzn/alexa-skills-kit-java"><img src="https://github.com/konpa/devicon/raw/master/icons/java/java-original.svg?sanitize=true" width="25px" /> Alexa Skills Kit SDK for Java</a>
+<a href="https://github.com/alexa/alexa-skills-kit-java"><img src="https://github.com/konpa/devicon/raw/master/icons/java/java-original.svg?sanitize=true" width="25px" /> Alexa Skills Kit SDK for Java</a>
```
Comment (tianrenz) [inline README.md]: The correct link should be https://github.com/alexa/alexa-skills-kit-sdk-for-java

File: README.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ The ASK SDK v2 for Node.js makes it easier for you to build highly engaging skil
 |[ask-sdk-s3-persistence-adapter](https://github.com/alexa/alexa-skills-kit-sdk-for-nodejs/tree/2.0.x/ask-sdk-s3-persistence-adapter)| [![npm](https://img.shields.io/npm/v/ask-sdk-s3-persistence-adapter.svg)](https://www.npmjs.com/package/ask-sdk-s3-persistence-adapter) [![npm](https://img.shields.io/npm/dt/ask-sdk-s3-persistence-adapter.svg)](https://www.npmjs.com/package/ask-sdk-s3-persistence-adapter)|
 |[ask-sdk-v1adapter](https://github.com/alexa/alexa-skills-kit-sdk-for-nodejs/tree/2.0.x/ask-sdk-v1adapter)|[![npm](https://img.shields.io/npm/v/ask-sdk-v1adapter.svg)](https://www.npmjs.com/package/ask-sdk-v1adapter) [![npm](https://img.shields.io/npm/dt/ask-sdk-v1adapter.svg)](https://www.npmjs.com/package/ask-sdk-v1adapter)|
 
-## Alexa Features Supported by SDK
+## Alexa Features supported by SDK
```
Comment (tianrenz) [inline README.md]: This is intended to be in upper case.

**Review by tianrenz — COMMENTED** [review]
Thanks for your help with the SDK! I've left some comments that needs your attention. Once those are fixed we can review again!

File: README.md (hunk 2 above)
Comment (LorenzNickel) [reply README.md]: Makes sense too, I just compared it with [this readme](https://github.com/alexa/alexa-skills-kit-sdk-for-java/blob/2.0.x/Readme.md) and I thought lower case would fit better, but you got me

Comment (tianrenz) [reply README.md]: Maybe we should update that readme instead. :D

Comment (LorenzNickel) [reply README.md]: Yeah, I have a [Pull-Request there](https://github.com/alexa/alexa-skills-kit-sdk-for-java/pull/182), I'll include it

**Comment by LorenzNickel** [comment]
Thanks for your review, should be fixed now

**Review by tianrenz — APPROVED** [review]

**Pull request merged**